python/rpdk/java/templates/init/guided_aws/Translator.java

```diff
@@ -0,0 +1,114 @@
+package {{ package_name }};
+
+import com.google.common.collect.Lists;
+import software.amazon.awssdk.awscore.AwsRequest;
+import software.amazon.awssdk.awscore.AwsResponse;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * This class is a centralized placeholder for
+ *  - api request construction
+ *  - object translation to/from aws sdk
+ *  - resource model construction for read/list handlers
+ */
+
+public class Translator {
+
+  /**
+   * Request to create a resource
+   * @param model resource model
+   * @return awsRequest the aws service request to create a resource
+   */
+  static AwsRequest translateToCreateRequest(final ResourceModel model) {
+    final AwsRequest awsRequest = null;
+    // TODO: construct a request
+    // e.g. https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/blob/master/aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/Translator.java#L39
+    return awsRequest;
+  }
+
+  /**
+   * Request to read a resource
+   * @param model resource model
+   * @return awsRequest the aws service request to describe a resource
+   */
+  static AwsRequest translateToReadRequest(final ResourceModel model) {
+    final AwsRequest awsRequest = null;
+    // TODO: construct a request
+    // e.g. https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/blob/master/aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/Translator.java#L20
+    return awsRequest;
+  }
+
+  /**
+   * Translates resource object from sdk into a resource model
+   * @param awsResponse the aws service describe resource response
+   * @return model resource model
+   */
+  static ResourceModel translateFromReadResponse(final AwsResponse awsResponse) {
+    // e.g. https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/blob/master/aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/Translator.java#L58
+    return ResourceModel.builder()
+        //.primaryIdentifier(response.primaryIdentifier())
```

Comment:
there isn't actually a property called `primaryIdentifier` that is meant to be set.  I would be confused by this.  I think you should just remove this line

Reply:
We don't know what the properties are, so this is just suggestive. Can remove though.